src/lib.rs

```diff
@@ -3,6 +3,11 @@ extern crate libloading;
 pub use libloading::{Library, Symbol};
 
 #[cfg(feature = "dlopen")]
+#[deprecated(
+    since = "0.5", // TODO: change to version of next release after 0.4.2
```

Comment:
Honestly, the next release will be a breaking release, so let's just remove the `dlopen` feature on dlib altogether. There is no point in keeping some hybrid behavior, cargo won't make an automatic update anyway.